api4/channel.go

```diff
@@ -79,8 +81,17 @@ func (api *API) InitChannel() {
 }
 
 func createChannel(c *Context, w http.ResponseWriter, r *http.Request) {
+	postBody, readErr := ioutil.ReadAll(r.Body)
+	if readErr != nil {
+		return
+	}
+	defer r.Body.Close()
```

Comment:
Handlers don't need to close the body: https://pkg.go.dev/net/http#:~:text=//%20The%20Server%20will%20close%20the%20request%20body.%20The%20ServeHTTP%0A%09//%20Handler%20does%20not%20need%20to.